### DISCUSSION TITLE
Is there a quick way to try out an branch from an extension's  repo?

### DISCUSSION THREAD

**Discussion by Brian-Ruff-CardinalPeak** [microsoft/vscode-discussions#741] [Extension Development QnA] [answered]
I am looking to try out a proposed fix to an extension and was wondering how best to easily deploy that branch.  Can you do something like layout the sources in the .vscode/extensions folder?

**Comment by alefragnani** [comment, marked as answer]
Hi @Brian-Ruff-CardinalPeak ,

I suppose you want to try out the branch locally, right?

If that's the case, following the regular [Your First Extension](https://code.visualstudio.com/api/get-started/your-first-extension) tutorial would be enough, and once you start the debugging process, the **Extension Development Host** window will load your version/branch, instead the installed version.

But first, you should look for _contributing_ details on the extension's repo you are trying to propose a fix, to find out if the extension has some additional requirements to be built.

Hope this helps
